### PR TITLE
Enable SELinux rules for RHEL9

### PR DESCRIPTION
**Pull request by matejak**
There were no changes in SELinux that would affect our rules, and SELinux booleans also remain the same.

There were just small fixes in the rule description.

### CODE REVIEW

**Comment by JAORMX** [comment]
```
 INFO[2021-05-06T10:24:52Z] level=info msg=Creating infrastructure resources... 
INFO[2021-05-06T10:24:52Z] level=error                                  
INFO[2021-05-06T10:24:52Z] level=error msg=Error: error waiting for Route53 Hosted Zone (Z08934382BDGFRH7AWB2I) creation: timeout while waiting for state to become 'INSYNC' (timeout: 15m0s) 
INFO[2021-05-06T10:24:52Z] level=error                                  
INFO[2021-05-06T10:24:52Z] level=error msg=  on ../tmp/openshift-install-010907806/route53/base.tf line 22, in resource "aws_route53_zone" "new_int": 
INFO[2021-05-06T10:24:52Z] level=error msg=  22: resource "aws_route53_zone" "new_int" { 
INFO[2021-05-06T10:24:52Z] level=error                                  
INFO[2021-05-06T10:24:52Z] level=error                                  
INFO[2021-05-06T10:24:52Z] level=fatal msg=failed to fetch Cluster: failed to generate asset "Cluster": failed to create cluster: failed to apply Terraform: failed to complete the change 
```
/retest

Infra failure

File: linux_os/guide/services/docker/docker_selinux_enabled/rule.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: rhel7,rhel8
+prodtype: rhel7,rhel8,rhel9
```
Comment (jan-cerny) [inline linux_os/guide/services/docker/docker_selinux_enabled/rule.yml:3]: AFAIK Docker is only in RHEL 7. Moreover the rule has a RHEL7 - only OVAL. I suggest restricting this prodtype to rhel7.
```suggestion
prodtype: rhel7
```

**Pull request merged**